openmls/src/messages/proposals_in.rs

```diff
@@ -70,6 +70,8 @@ pub enum ProposalIn {
     //             was moved to `draft-ietf-mls-extensions-00`.
     #[tls_codec(discriminant = 8)]
     AppAck(AppAckProposal),
+    DeviceAdd(AddProposalIn),
+    DeviceRemoveProposal(RemoveProposal),
```

Comment:
Any reason this one isn't just `DeviceRemove` for consistency?

Reply:
No reason... this should be `DeviceRemove` 🙂